Documentation/api/ClientHowto.md

```diff
@@ -37,7 +37,7 @@ The `--log-to-file` and `--log-to-fd` options can be used to redirect the "API s
 
 ## Controlling the backend
 
-Once you have a running headless instance you can connect to it and start sending commands. Delve's protocol is built on top of the [JSON-RPC](http://json-rpc.org) specification.
+Once you have a running headless instance you can connect to it and start sending commands. Delve's protocol is built on top of the [JSON-RPC specification](https://www.jsonrpc.org/specification).
```

Comment:
We use JSON-RPC 1.0 so this should be https://www.jsonrpc.org/specification_v1

Reply:
oops, updated